src/lib/viewers/controls/media/PlayPauseToggle.tsx

```diff
@@ -1,22 +1,45 @@
 import React from 'react';
 import noop from 'lodash/noop';
-import IconPause24 from '../icons/IconPause24';
 import IconPlay24 from '../icons/IconPlay24';
+import IconPause24 from '../icons/IconPause24';
+import IconForward24 from '../icons/IconArrowCurveForward24';
+import IconBack24 from '../icons/IconArrowCurveBack24';
 import MediaToggle from './MediaToggle';
 import './PlayPauseToggle.scss';
 
 export type Props = {
     isPlaying?: boolean;
     onPlayPause: (isPlaying: boolean) => void;
+    movePlayback?: (forward: boolean, duration: number) => void;
 };
 
-export default function PlayPauseToggle({ isPlaying, onPlayPause = noop }: Props): JSX.Element {
+export default function PlayPauseToggle({ isPlaying, onPlayPause = noop, movePlayback = noop }: Props): JSX.Element {
     const Icon = isPlaying ? IconPause24 : IconPlay24;
     const title = isPlaying ? __('media_pause') : __('media_play');
+    const skipForwardTitle = __('media_skip_forward');
+    const skipBackwardTitle = __('media_skip_backward');
+
+    const moveForward = (): void => {
+        movePlayback(true, 5);
+    };
+
+    const moveBackward = (): void => {
+        movePlayback(false, 5);
+    };
 
     return (
-        <MediaToggle className="bp-PlayPauseToggle" onClick={(): void => onPlayPause(!isPlaying)} title={title}>
-            <Icon />
-        </MediaToggle>
+        <>
+            <MediaToggle className="bp-PlayPauseToggle" onClick={moveForward} title={skipForwardTitle}>
+                <IconForward24 />
+            </MediaToggle>
+
+            <MediaToggle className="bp-PlayPauseToggle" onClick={(): void => onPlayPause(!isPlaying)} title={title}>
+                <Icon />
```

Comment:
I know it's a little out of scope, but `Icon` is really generic, can this be PlayPauseIcon or something specific